skore/src/skore/sklearn/train_test_split/warning/time_based_column_warning.py

```diff
@@ -0,0 +1,63 @@
+"""'Time-based column' warning.
+
+This warning is shown when ``X`` contains a time-based column.
```

Comment:
Please can you precise that only pandas/polars dataframe are supported?

Reply:
Will do